ledger-test-tool/suites/deps.bzl

```diff
@@ -16,8 +16,7 @@ def deps(lf_version):
         "//test-common:carbonv2-tests-%s.scala" % lf_version,
         "//test-common:carbonv3-tests-%s.scala" % lf_version,
     ]
-    model_tests = ["//test-common:modelext-tests-%s.scala" % lf_version]
```

Comment:
Why can this be removed?

Reply:
It's not used anymore. It was used only for usage with interfaces which should be handled separately in the integration tests that you are implementing